### DISCUSSION TITLE
How to trim a transparent background in the bottom of an image?

### DISCUSSION THREAD

**Discussion by yenhle16** [ImageMagick/ImageMagick#2262] [Help]
How to trim a transparent background in the bottom of an image?
Thank you!

**Comment by fmw42** [comment]
What is your ImageMagick version and platform?  Can you post an example image?

use -splice to add two single rows of contrasting colors at the top. then use -trim, then chop off the top row.  See -splice and -trim and -chop at https://imagemagick.org/Usage/crop/  In particular see https://imagemagick.org/Usage/crop/#trim_oneside

`convert image -gravity north  -background white -splice 0x1  -background black -splice 0x1 -trim  +repage -chop 0x1  result`
